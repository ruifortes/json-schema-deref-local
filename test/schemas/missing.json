{
  "description": "Just a basic schema.",
  "title": "Basic Object",
  "type": "object",
  "definitions": {
    "foo": {
      "description": "foo property",
      "readOnly": true,
      "type": "number"
    },
    "bar": {
      "description": "bar property",
      "type": "boolean"
    }
  },
  "properties": {
    "foo": {
      "$ref": "#/definitions/foo"
    },
    "bar": {
      "$ref": "#/definitions/bar"
    },
    "stuff": {
      "$ref": "#/definitions/stuff"
    },
    "email": {
      "$ref": "common-definitions.json#/definitions/missing"
    }
  }
}