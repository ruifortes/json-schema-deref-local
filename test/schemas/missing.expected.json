{
  "description": "Just a basic schema.",
  "title": "Basic Object",
  "type": "object",
  "definitions": {
    "foo": {
      "description": "foo property",
      "readOnly": true,
      "type": "number"
    },
    "bar": {
      "description": "bar property",
      "type": "boolean"
    }
  },
  "properties": {
    "foo": {
      "description": "foo property",
      "readOnly": true,
      "type": "number"
    },
    "bar": {
      "description": "bar property",
      "type": "boolean"
    },
    "stuff": {
      "$ref": "#/definitions/stuff"
    },
    "email": {
      "$ref": "common-definitions.json#/definitions/missing"
    }
  }
}